### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arps18**
Added Flutter installation links and other useful links.

### CODE REVIEW

**Comment by harshil1712** [comment]
Add a section in the begning where it describes the App

File: reps_event_app/README.md [hunk 1]
```diff
@@ -14,3 +14,99 @@ A few resources to get you started if this is your first Flutter project:
...
+* [Install Flutter](https://flutter.dev/get-started/)
+* [Flutter documentation](https://flutter.dev/docs)
+* [Development wiki](https://github.com/flutter/flutter/wiki)
+* [Contributing to Flutter](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md)
+
+For announcements about new releases and breaking changes, follow the
+[flutter-announce@googlegroups.com](https://groups.google.com/forum/#!forum/flutter-announce)
+mailing list.
+
+## About Flutter
```
Comment (harshil1712) [inline reps_event_app/README.md:37]:
```suggestion
## About Flutter
```
It's okay to not have this section

File: reps_event_app/README.md [hunk 2]
```diff
@@ -14,3 +14,99 @@ A few resources to get you started if this is your first Flutter project:
...
+
+We think Flutter will help you create beautiful, fast apps, with a productive,
+extensible and open development model.
+
+### Beautiful apps
```
Comment (harshil1712) [inline reps_event_app/README.md:42]:
```suggestion
### Beautiful apps
```
This section is also not required

**Review by harshil1712 — CHANGES_REQUESTED** [review]
Please read the issue properly. The REAMDE file should contain recipe on how to install Flutter, setup the developer environment, and provide some references.  All the other details aren't required.

**Comment by obitodarky** [comment]
Closing this since we need the Readme file in the `dev` branch and not in the `master` branch. 
@arps18  please make the changes requested by @harshil1712 and send a PR to the `dev` branch. Thanks

**Pull request closed without merging**